purchases/src/main/kotlin/com/revenuecat/purchases/Purchase.kt

```diff
@@ -0,0 +1,77 @@
+package com.revenuecat.purchases
+
+import android.app.Activity
+import com.revenuecat.purchases.models.GoogleProrationMode
+import com.revenuecat.purchases.models.PurchasingData
+import com.revenuecat.purchases.models.StoreProduct
+import com.revenuecat.purchases.models.SubscriptionOption
+
+// TODO BC5 naming
+open class Purchase(builder: Builder) {
+
+    val isPersonalizedPrice: Boolean
+    val oldProductId: String?
+    val googleProrationMode: GoogleProrationMode
+    internal val purchasingData: PurchasingData
+    internal val activity: Activity
```

Comment:
Friendly reminder to make sure this class' API looks good in Java. So far I think it should be fine

Reply:
it's been nice having the API tester, for some reason i held off on pushing that bc i wanted it in it's own commit 😓 but it's looking good so far!